examples/ad9545_example.py

```diff
@@ -0,0 +1,23 @@
+import adijif
+import pprint
+
+clk = adijif.ad9545(solver="gekko")
+
+input_refs = [1, 10e6, 0, 0]
+
+output_clocks = [30720000, 0, 0 , 0, 0, 0, 0 ,0 ,0, 0]
```

Comment:
Do you have to put zeros? or can you just have a smaller list?

Reply:
The zeros are necessary or i could change the method to accept an array of tuples (input_number, rate).

Reply:
I would recommend accepting only clocks the user cares about aka not zero pad. This is how the other clock chips function. But also limit the user to 10 output clocks